activiti-spring-connector/src/main/java/org/activiti/spring/connector/autoconfigure/ConnectorAutoConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.connector.autoconfigure;
+
+import org.activiti.spring.connector.ConnectorService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConnectorAutoConfiguration {
+
+    @Bean
```

Comment:
@ffazzini instead of returning a bean, this guy can add a @ComponentScan here with it's own package. Does it make sense? we need to follow that convention every time that we have a @Component or a @Service.

Reply:
Don't understand, can you code what you mean? Thanks.
Talking with @ryandawsonuk earlier i think you guys have in mind to refactor the existing design you have
which has a mix of autoconfiguration and componentscans?

Reply:
@ffazzini 👍 sorry for the delay, that was exactly what I meant!